src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -2,16 +2,23 @@ package racingcar.domain
 
 import racingcar.domain.movingstrategy.MovingStrategy
 
-class Car(private val movingStrategy: MovingStrategy) {
+class Car(
+    private val movingStrategy: MovingStrategy,
+    private val name: CarName,
+) {
     var position: Int = DEFAULT_POSITION
         private set
 
+    constructor(movingStrategy: MovingStrategy, name: String) : this(movingStrategy, CarName(name))
+
     fun move() {
         if (movingStrategy.isMovable()) {
             position++
         }
     }
 
+    fun getCurrentInfo() = CarInfo(name.value, position)
```

Comment:
제가 의도했던 부분은 function 이 아닌 단순 value 로 접근하는 것을 말씀드리고싶었습니다 :) 
```suggestion
    val getCurrentInfo(): CarInfo
       get() = CarInfo(name.value, position)
```